shop/src/Components/Price/index.jsx

```diff
@@ -1,20 +1,81 @@
 import "./Price.scss";
-const Price = () => {
+import { useState, useEffect } from "react";
+import { dolorFormat } from "../../util/dolarFormat";
+
+const Price = ({ shoeItems, step, handleOnPriceRange }) => {
+  const [minPrice, setMinPrice] = useState();
+  const [maxPrice, setMaxPrice] = useState();
+
+  const [minValue, setMinValue] = useState(0);
+  const [maxValue, setMaxValue] = useState(30000);
+
```

Comment:
```ruby
const [minPrice, setMinPrice] = useState();
const [maxPrice, setMaxPrice] = useState();
```
and

```ruby
const [minValue, setMinValue] = useState(0);
const [maxValue, setMaxValue] = useState(30000);
```

all of these states can just be 2 state
`const [value, setValue] = useState({min: 0, max: 30000});`
`const [price, setPrice] = useState({min: 0, max: 0});`